From: Jakub Adam <jakub.adam@ktknet.cz>
Date: Sun, 1 Apr 2012 21:33:35 +0200
Subject: tests-init-ant-project

Apache Ant 1.8.2 will not automatically initialize a new Project.
This fixes FTBFS #666612.
---
 .../eclipse/jgit/ant/tasks/GitCloneTaskTest.java   |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/org.eclipse.jgit.ant.test/src/org/eclipse/jgit/ant/tasks/GitCloneTaskTest.java b/org.eclipse.jgit.ant.test/src/org/eclipse/jgit/ant/tasks/GitCloneTaskTest.java
index c7b949c..2da9d2a 100644
--- a/org.eclipse.jgit.ant.test/src/org/eclipse/jgit/ant/tasks/GitCloneTaskTest.java
+++ b/org.eclipse.jgit.ant.test/src/org/eclipse/jgit/ant/tasks/GitCloneTaskTest.java
@@ -64,6 +64,7 @@
 	@Before
 	public void before() {
 		project = new Project();
+		project.init();
 		enableLogging();
 		project.addTaskDefinition("git-clone", GitCloneTask.class);
 		task = (GitCloneTask) project.createTask("git-clone");
